lib/posix/options/Kconfig.pipe

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024, Bruce Rosier
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menu "POSIX Pipe Options"
+
+config POSIX_PIPE
+	bool"POSIX pipe support"
+	default y
+	depends on POSIX_API
+
+config POSIX_NUM_PIPES
+	int "Number of available pipes"
+	default 2
+	depends on POSIX_API
```

Comment:
Please remove this dependency. `CONFIG_POSIX_API` will likely be deprecated soon. 
```suggestion
```